contessa/executor.py

```diff
@@ -79,24 +42,12 @@ def compose_where_condition(self, rule):
         else:
             return ""
 
-    def filter_df(self, rule):
-        """
-        Filter the `raw_df` to be give it to apply method. Rule attribute `time_filter` filters
-        only data that were updated/created/confirmed in last 30 days.
-        :return: pd.Dataframe
-        """
-        if rule.time_filter:
-            raise NotImplementedError(
-                "time_filter is not supported with PandasExecutor"
-            )
-        return self.raw_df
-
     def execute(self, rule):
         """
         Main entrypoint for Executor class. Composing kwargs specific for the executor
         (e.g. PandasExecutor) and get the results from rule.
         :param rule: Rule
-        :return: pd.Series
+        :return: TODO
```

Comment:
Why is this not clear? Can we just use generic iterable?

Reply:
I wasnt sure what will be the return type when I was rewriting this file and forgot to fix it later.

Reply:
fixed